src/applications/pact-act/constants/results-set-1-page-1-dynamic-content.jsx

```diff
@@ -1,13 +1,13 @@
 import React, { Fragment } from 'react';
 import { RESPONSES, SHORT_NAME_MAP } from './question-data-map';
-import { locationList as BurnPit21Locations } from '../containers/questions/burn-pit/BurnPit-2-1';
 import { locationList as BurnPit211Locations } from '../containers/questions/burn-pit/BurnPit-2-1-1';
 import { locationList as BurnPit212Locations } from '../containers/questions/burn-pit/BurnPit-2-1-2';
+import { locationList as BurnPit213Locations } from '../containers/questions/burn-pit/BurnPit-2-1-3';
```

Comment:
Results page 1.1 shows dynamic content depending on what Burn Pit questions you selected "Yes" for. Since we added a new Burn Pit question in front of the other 3, I updated the short names here so the content matched up correctly.